Bigtable/src/Instance.php

```diff
@@ -0,0 +1,331 @@
+     *     @type int $type Possible values are represented by the following constants:
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`,
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT` and
+     *           `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *           **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     * }
+     *
+     * @return LongRunningOperation<Instance>
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function create(array $clusterMetadataList, array $options = [])
+    {
+        if (empty($clusterMetadataList)) {
+            throw new \InvalidArgumentException('At least one clusterMetadata must be passed');
+        }
+        $projectName = InstanceAdminClient::projectName($this->projectId);
+        $displayName = isset($options['displayName']) ? $options['displayName'] : $this->id;
+        $labels = isset($options['labels']) ? $options['labels'] : [];
+        $type = isset($options['type']) ? $options['type'] : self::INSTANCE_TYPE_UNSPECIFIED;
+
+        $clustersArray = [];
+        foreach ($clusterMetadataList as $value) {
+            if (!isset($value['clusterId'])) {
+                throw new \InvalidArgumentException('Cluster id must be set.');
+            }
+            $this->validate($value['clusterId'], 'cluster');
+            $clusterId = $value['clusterId'];
+            if (!isset($value['locationId'])) {
+                throw new \InvalidArgumentException('Location id must be set.');
+            }
+            $this->validate($value['locationId'], 'location');
+            $locationId = $value['locationId'];
+            $value['location'] = InstanceAdminClient::locationName($this->projectId, $locationId);
+            $value['defaultStorageType'] = isset($value['storageType'])
+                ? $value['storageType']
+                : self::STORAGE_TYPE_UNSPECIFIED;
+
+            if ($type === self::INSTANCE_TYPE_DEVELOPMENT) {
+                unset($value['serveNodes']);
+            } elseif (!isset($value['serveNodes']) || $value['serveNodes'] <= 0) {
+                throw new \InvalidArgumentException('When creating Production instance, serveNodes must be > 0');
+            }
+            // `$clustersArray` must be keyed by the cluster ID.
+            $clustersArray[$clusterId] = $value;
+        }
+        $operation = $this->connection->createInstance([
+            'parent' => $projectName,
+            'instanceId' => $this->id,
+            'instance' => [
+                'displayName' => $displayName,
+                'type' => $type,
+                'labels' => $labels
+            ] + $options,
+            'clusters' => $clustersArray
+        ]);
+
+        return $this->resumeOperation($operation['name'], $operation);
```

Comment:
We should just return the operation.  Resume doesn't do anything in the context of Cloud Bigtable.